generator/vogtle_data_generator.py

```diff
@@ -51,23 +50,14 @@ def __init__(self, directory=None):
             logging.error("No directory provided, exiting..")
             sys.exit(1)
 
+        os.makedirs(directory, exist_ok=True)
```

Comment:
The [pathlib](https://docs.python.org/3.7/library/pathlib.html) builtin library has a `Path` object that is handy.  One of its methods will create directories too.